crates/sui-framework/packages/sui-framework/sources/tx_context.move

```diff
@@ -75,6 +75,10 @@ fun ids_created(_self: &TxContext): u64 {
 }
 native fun native_ids_created(): u64;
 
+#[allow(unused_function)]
+// native function to retrieve gas price, currently not exposed
+native fun native_rgp(): u64;
```

Comment:
just a note, for the public function, we probably want to spell this out as `reference_graph_price`